src/backend/mmap.rs

```diff
@@ -0,0 +1,151 @@
+extern crate memmap;
+extern crate failure;
```

Comment:
Please do no put `extern crate` statements in any other file than `src/lib.rs`, you can still access them normally from there